### ISSUE TITLE
gvp:broader is geen subproperty meer van skos:broader...

### ISSUE THREAD

**Issue by dieuska** [OnroerendErfgoed/skosprovider_getty#18]
Doordat gvp:broader geen subproperty van skos:broader meer is, wordt gvp:broader niet meer als broader opgenomen bij het Concept. Hierdoor faalt er ineens een test.
gvp:broader-->http://vocab.getty.edu/ontology#broader

Zie bvb "test_get_by_id_concept":
http://vocab.getty.edu/aat/300007466 heeft een gvp:broaderPrefered (http://vocab.getty.edu/aat/300007391), maar deze wordt nu niet meer als broader gekoppeld aan het Concept

Dit lijkt mij een wijziging aan de kant van Getty..., want deze test was ok bij een vorige push.

**Comment by koenedaele** [comment]
Is there also a problem with skos:narrower? I can't seem to find either of those anymore through the vocab browser.

**Comment by dieuska** [comment]
indeed: gvp:narrower isn't a subproperty anymore of skos:narrower...
http://vocab.getty.edu/ontology#narrower
*One possibility mabe is that we define at initialisation that gvp:narrower/gvp:broader has to be interpreted as subproperties of skos:narrower/broader.
*Or we can also rewrite the method that is uses gvp:narrower/gvp:broader directly to get our narrower/broader Concepts.

**Comment by dieuska** [comment]
same thing for SKOS:related.... 
There are no properties in Getty who are a subclass of SKOS:related, so according to the current code the resulting Concept will allways have a empty related list.
So writing seperate methods for finding broader/narrower/related according to the specific getty-logic would be an option. 
In these methods we define that the gvp:broader would be interpreted as the broader Concepts, gvp:narrower would be interpreted as the narrower Concepts
gvp:aat2000_related_to (http://vocab.getty.edu/ontology#anchor-1260930258) has to be interpreted as the related Concepts for aat, ...

**Comment by koenedaele** [comment]
I think it works like this. A gvp:subject is a used for both concepts and collections. If you download the RDF for "churches (buildings)", you can see that it's both a skos:concept and a gvp:subject. (And also a gvp:concept, but that doesn't really matter here).

Since a gvp:subject can be both a concept and a collection, the relation gvp:broaderPreferred can be between two concepts, two collections or a concept and a collection. Depending on the two types, they'll be different. So if we have A (skos:concept) gvp:broaderPreferred (and thus gvp:broader) B (skos:concept), it's a broader relation. If we have A (skos:concept) gvp:broaderPreferred B (skos:collection), it's a member_of relation. If we have A (skos:collection) gvp:broaderPreferred B (skos:concept), it's a superordinate concept relation. If we have A (skos:collection) gvp:broaderPreferred B (skos:collection), it's a member_of relation.

**Comment by koenedaele** [comment]
About the related relations: let's forget about those. It's already compliated enough as it is and there's not too much added value in them.

**Comment by koenedaele** [comment]
Although I don't really understand why there are iso-thes:subordinateArray predicates present and not skos:narrower predicates.

**Comment by koenedaele** [comment]
Moving this to 0.3.0. 

I think what I wrote above is correct. But we'll also have to be able to handle the narrower relations, since they are not present in the current individual SKOS files. Altough it's possible the GRI team will include those in a next version.
